modin/numpy/arr.py

```diff
@@ -294,7 +282,22 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
             args += [
                 input._query_compiler if hasattr(input, "_query_compiler") else input
             ]
-        return array(_query_compiler=new_ufunc(*args, **kwargs), _ndim=out_ndim)
+        out_kwarg = kwargs.get("out", None)
+        if out_kwarg is not None:
+            # If `out` is a modin.numpy.array, `kwargs.get("out")` returns a 1-tuple
```

Comment:
but what if `out` is _not_ a `modin.numpy.array`?.. shouldn't we `isinstance()` check it instead of `is not None`?

Reply:
That's handled by the function `fix_dtypes_and_determine_return`. It calls `check_can_broadcast_to_output`, which first checks if the out variable is a modin.numpy.array, and errors if it is not!

Reply:
TBH I still don't understand why do we pass a variable using a 1-element tuple... feels super-weird. Could you pls explain?

Reply:
Sorry - to clarify, we don't do that - that's just how NumPy handles the arguments. Say someone calls `numpy.tanh(modin.numpy.array, *args, **kwargs)` - the following happens:
1. NumPy recognizes that our `modin.numpy.array` object has an `__array_ufunc__` method.
2. NumPy translates `numpy.tanh` to `modin.numpy.array.__array_ufunc__(numpy.tanh, "__call__", *args, **kwargs)`
Our `out` argument is inside the `**kwargs`, but the packaging (it being inside a 1-tuple) is handled by NumPy, so I just unpack it if its 1-tuple.